snmp/README.md

```diff
@@ -4,211 +4,102 @@
+SNMP uses sysOIDs (System Object Identifiers) to uniquely identify devices, and OIDs (Object Identifiers) to uniquely identify managed objects. OIDs follow a hierarchical tree pattern: under the root is ISO, which is numbered 1. The next level is ORG and numbered 3 and so on, with each level being separated by a `.`.
 
-A MIB (Management Information Base) acts as a translator between OIDs and human readable names, and organizes a subset of the hierarchy. Because of the way the tree is structured, most SNMP values start with the same set of objects: 1.3.6.1.1 for MIB-2 which is a standard that holds system information like uptime, interfaces, network stack, and 1.3.6.1.4.1 which holds vendor specific information.
+A MIB (Management Information Base) acts as a translator between OIDs and human readable names, and organizes a subset of the hierarchy. Because of the way the tree is structured, most SNMP values start with the same set of objects:
+
+* `1.3.6.1.1`: (MIB-II) A standard that holds system information like uptime, interfaces, network stack.
+* `1.3.6.1.4.1`: A standard that holds vendor specific information.
 
 ## Setup
 ### Installation
 
 The SNMP check is included in the [Datadog Agent][1] package. No additional installation is necessary.
 
 ### Configuration
-#### Host
-
-Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
-
-The SNMP check doesn't collect anything by default. Specify metrics to collect by updating your `snmp.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][2]. See the [sample snmp.d/conf.yaml][3] for all available configuration options.
-
-##### SNMP v1-v2 configuration
-
-```yaml
-init_config:
-  ## @param mibs_folder - string - required
-  # mibs_folder: <PATH_TO_ADDITIONAL_MIBS>
 
-  ## @param global_metrics - list of elements - (required if none at the instance level)
-  # global_metrics:
-  ## - MIB: <MIB_NAME>
-  ##   symbol: <SYMBOL>
+<div class="alert alert-warning">
+<b>Note</b>: The following features are in beta.
+</div>
 
-instances:
-    ## @param ip_address - string - required
-  - ip_address: localhost
-
-    ## @param port - integer - required
-    ## Default SNMP port.
-    #
-    port: 161
-
-    ## @param community_string - string - optional
-    # community_string: public
-
-    ## @param snmp_version - integer - optional - default: 2
-    # snmp_version: 2
-
-    ## @param metrics - list of elements - (required if none at the init_config level)
-    metrics:
-    - MIB: <MIB_NAME>
-      symbol: <SYMBOL>
-    - OID: <OID>
-      name: <OID_NAME>
-      metric_tags:
-        - <TAG>
-```
+The Datadog SNMP check scans a provided subnet, auto discovers network devices, and collects metrics using Datadog's sysOID mapped device profiles.
```

Comment:
```suggestion
The Datadog SNMP check scans a provided subnet, Autodiscovers network devices, and collects metrics using Datadog's sysOID mapped device profiles.
```

let's also add a link to the Autodiscovery page on `Autodiscovers` since i believe that's the first mention of AD on this page

Reply:
It's a different autodiscovery than the agent integration autodiscovery